package/yast2-services-manager.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Wed May  8 15:20:54 UTC 2019 - Stefan Hundhammer <shundhammer@suse.com>
+
+- Adapted systemd states to upstream changes:
+  New stop-watchdog, no more stop-sigabrt
+- 4.1.15
```

Comment:
This is SP2 only, or a backport to SP1 is planned? For SP2 we should bump the version to 4.2.0...

Reply:
fine with me; bumping the version then.